src/ralph/domains/models/domains.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from dj.choices import Choices
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.assets.models import BaseObject
+from ralph.assets.models.assets import BusinessSegment
+from ralph.back_office.models import AssetHolder
+from ralph.lib.mixins.models import NamedMixin, TimeStampMixin
+from ralph.lib.permissions import PermByFieldMixin
+
+
+class DomainRegistrant(
+    PermByFieldMixin,
+    NamedMixin,
+    TimeStampMixin,
+    models.Model,
+):
+    pass
+
+
+class DomainStatus(Choices):
+    _ = Choices.Choice
+
+    active = _('Active')
+    pending_lapse = _('Pending lapse')
+    pending_transfer = _('Pending transfer away')
+    lapsed = _('Lapsed (inactive)')
+    transfered_away = _('Transfered away')
+
+
+class Domain(BaseObject):
+    name = models.CharField(
+        verbose_name=_('Domain name'),
+        help_text=_('Full domain name'),
+        unique=True,
+        max_length=255
+    )
+    domain_status = models.PositiveIntegerField(
+        choices=DomainStatus(),
+        default=DomainStatus.active.id,
+    )
+    business_segment = models.ForeignKey(
+        BusinessSegment, blank=True, null=True,
+        help_text=_("Business segment for a domain")
+    )
+    business_owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
```

Comment:
i don't remember if i've asked it before but what do you think about many-to-many here? will there be one (and only one) contact person for domain? In case of services we allow for many owners.